.github/workflows/test-next-release-prs.yml

```diff
@@ -0,0 +1,59 @@
+# Description: This workflow runs unit + e2e tests on PRs targeting
```

Comment:
Copy and pasted from `test-rwa.yml`. In the future, we can definitely improve this exprience of having to copy paste workflows for every non-trivial feature branch. I'll take a look after all the feature works are done.